### PR TITLE
Implementation Of Tower of Hanoi in C#

### LINKED ISSUE #2479: Tower of Hanoi
- [x] C
- [x] C++
- [x] C# - @biekymahato
- [ ] CoffeeScript
- [x] Dart - @disha2sinha
- [x] Go - @rbjoshi1309
- [x] Java
- [x] JS
- [x] Kotlin - @chandbud5
- [x] Php - @chandrikadeb7
- [x] Python
- [x] Readme
- [x] Ruby - @HardevKhandhar
- [ ] Rust
- [x] TypeScript - @vyom153069

### PR DESCRIPTION
**Pull request by biekymahato**
Fixes #2479 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Languages Used:
- c#

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by biekymahato** [comment]
@asha15 have a look

File: Tower Of Hanoi (c#)/Tower_Of Hanoi.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+C# recursive program to solve
+tower of hanoi puzzle */
+using System;
+
+class geek
+{
+
+    // C# recursive function to solve tower of hanoi puzzle
+    static void towerOfHanoi(int n, char from_rod,
+                             char to_rod, char aux_rod)
```
Comment (asha15) [inline Tower Of Hanoi (c#)/Tower_Of Hanoi.cs]: take this to the previous line

**Comment by asha15** [comment]
Also put this file inside Tower_Of_Hanoi folder. Do this change and let me knwo

**Comment by biekymahato** [comment]
@asha15 have a look. Done the changes.

**Comment by biekymahato** [comment]
@asha15  done all the changes

**Comment by biekymahato** [comment]
@asha15    Done

**Comment by biekymahato** [comment]
@asha15 done.

**Comment by asha15** [comment]
could you squash the commits ?

**Comment by raghav-dalmia** [comment]
Still, your PR has 4 commits. Squash them.

**Pull request merged**